bin/gce/install.sh

```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env bash
 # Based on gs://dataproc-initialization-actions/jupyter/jupyter.sh
+# Installs the following on the dataproc master: 
+#   conda
+#   jupyter
+#   npm
+#   nodejs
+#   maven
+#   docker
+#   google_compute_engine [pip]
+#   cigar [pip]
+# Installs the following on dataproc workers:
+#   conda
+#   cigar [pip]
```

Comment:
Im still unsure of why we need to install cigar. shouldn't this be installed when we install bdgenomics.mango?

Reply:
We'll probably have to distribute it in the PYSPARK_PYTHON to workers -- it looks like we currently don't do it. I think we should address this in a follow-up change.

https://www.cloudera.com/documentation/enterprise/5-5-x/topics/spark_python.html